kobo/settings/base.py

```diff
@@ -320,10 +320,12 @@ def __init__(self, *args, **kwargs):
 
 # Database
 # https://docs.djangoproject.com/en/1.7/ref/settings/#databases
+DATABASES = {}
+if 'KPI_DATABASE_URL' in env.ENVIRON:
+    DATABASES['default'] = env.db_url(default='KPI_DATABASE_URL')
```

Comment:
As I said on KoBoCAT PR, the syntax should be `env.db_url('KPI_DATABASE_URL', default="sqlite:///%s/db.sqlite3" % BASE_DIR)` 
If `DATABASE_URL` is not present, this will fail :-(